conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureUndertowRuntime.java

```diff
@@ -71,6 +84,12 @@ public Builder encodings(Encoding value) {
             return this;
         }
 
+        @CanIgnoreReturnValue
+        public Builder markedParamAction(MarkedParam value) {
```

Comment:
We may want to use a different interface here, that way we can add new methods to `MarkedParam` without breaking existing ConjureUndertowRuntime configurations.

Reply:
done